lib/puppet/network/http/api/indirected_routes.rb

```diff
@@ -111,24 +107,6 @@ def uri2indirection(http_method, uri, params)
 
   private
 
-  def do_http_control_exception(response, exception)
-    msg = exception.message
-    Puppet.info(msg)
-    response.respond_with(exception.status, "text/plain", msg)
-  end
-
-  def do_exception(response, exception, status=400)
-    if exception.is_a?(Puppet::Network::AuthorizationError)
-      # make sure we return the correct status code
-      # for authorization issues
-      status = 403 if status == 400
-    end
```

Comment:
This logic still needs to be handled somewhere, otherwise we'll return status 400 for authorization issues?